edgecast/resources/originv3/origin_httplarge_test.go

```diff
@@ -279,7 +266,7 @@ func TestExpandOrigins(t *testing.T) {
 		{
 			name:          "nil input",
 			input:         nil,
-			expectedPtr:   make([]*originv3.CustomerOriginRequest, 0),
+			expectedPtr:   make([]*OriginState, 0),
 			expectSuccess: false,
 		},
 	}
```

Comment:
Would be good to have a test for getOriginsDelta

Reply:
Refactored the delta code to make it unit testable.